Remove processes (#4271)
viamrobotics/docs: docs/dev/reference/changelog.md

<!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->

{{% changelog color="removed" title="Processes" date="2025-05-06" %}}

> Processes are no longer supported as of `viam-server` v0.74.0.

Author: ```suggestion
Processes are no longer supported for `viam-server` v0.74.0 and newer.
```

Reviewer: [q] I would expect a changelog to read something more like:

```suggestion
Processes were removed in `viam-server` v0.74.0.
```

I'm guessing there's a good reason why you chose a different wording, could you explain the thought process to me?
